api-specificatie/openapi.yaml

```diff
@@ -486,64 +429,39 @@ components:
           $ref: "#/components/schemas/Nummeraanduiding"
         woonplaatsen:
           $ref: "#/components/schemas/Woonplaats"
-    #JBO: Zoeksuggestie wordt hier als een redelijk generiek antwoord op een generieke zoekfuntie gebruikt. Is het te bedoelding dit te hergerbuiken in andere zoek-functie. 
-    #JBO: Wat hier "id" genoemd wordt het in de raadpleeg-functie de "key". Is dit verschil een gevolg van de keuze om dit generiek toe te kunnen passen ? 
-    ZoekSuggestie:
+        adresseerbaarobjecten:
+          oneOf:
+            - $ref: "#/components/schemas/Ligplaats"
+            - $ref: "#/components/schemas/Verblijfsobject"
+            - $ref: "#/components/schemas/Standplaats"
+          discriminator: 
+            propertyName: type
+    #JBO: Vraag is of we Keuzeitem moeten voorzien van Hal_links naar nummeraanduiding, adresseerbaarobject en adres.
+    Keuzeitem:
       type: object
       properties:
-        type:
-          description: 'Type van gevonden zoeksuggestie'
-          title: 'type'
-          type: string
         weergavenaam:
           description: 'Weergavenaam van de zoeksuggestie'
           title: 'weergavenaam'
           type: string
-        id:
-          description: 'Id van de zoeksuggestie'
-          title: 'id'
+        nummeraanduidingidentificatie:
+          description: 'Identificatie van de nummeraanduiding die aan dit adres gerelateerd is. '
+          title: 'nummeraanduidingidentificatie'
           type: string
-        score:
-          description: 'Score van de zoeksuggestie. Hoger is dichterbij de zoekterm'
-          title: 'score'
-          type: number
-
-    ZoekSuggestieResponse:
-      description: 'Lijst van suggestie op basis van een zoekstring'
-      title: ZoekSuggestieResponse
+        adresseerbaarobjectidentificatie:
+          description: 'Identificatie van het adresseerbaarobject dat aan dit adres gerelateerd is.'
+          title: 'adresseerbaarobjectidentificatie'
+          type: string
+    #JBO: Als we geen Hal-links opnemen in het Keuzeitem zouden we dit ook Keuzelijst kunnen noemen. 
```

Comment:
De suggestie in commentaar lijkt mij uitstekend.

Reply:
Even wachten op de conclusie. Issue is ingediend.